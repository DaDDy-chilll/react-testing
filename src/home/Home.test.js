import React from "react";
import HomePage from "./HomePage";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";

describe(`<HomePage/>`, () => {
  test("should render without crashing", () => {
    render(<HomePage />);
  });

  test("snapsho", () => {
    const tree = renderer.create(<HomePage />);
    expect(tree).toMatchSnapshot();
  });
});
