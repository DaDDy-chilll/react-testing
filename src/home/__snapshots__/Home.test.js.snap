// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage/> snapsho 1`] = `
<h2>
  Home
</h2>
`;
